hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/conf/HddsConfServlet.java

```diff
@@ -28,13 +28,12 @@
 import java.util.Map;
 import java.util.Properties;
 
-import com.google.common.base.Strings;
```

Comment:
It is good to remove the common Strings import. 👍

Reply:
Done!